Documentation/install.md

```diff
@@ -31,25 +24,35 @@ Follow instructions from the [releases](https://github.com/kinvolk/inspektor-gad
 * Download inspektor-gadget.zip:
   ![Download artifacts](github-actions-download-artifacts.png)
 * Finish the installation:
+
 ```
-$ unzip inspektor-gadget.zip
-$ chmod +x inspektor-gadget/inspektor-gadget
+$ unzip -p inspektor-gadget.zip | tar xvzf -
 $ sudo cp inspektor-gadget/inspektor-gadget /usr/local/bin/kubectl-gadget
+$ kubectl gadget version
 ```
 
-### From the sources:
+#### From the sources
 
 ```
 $ git clone https://github.com/kinvolk/inspektor-gadget.git
 $ cd inspektor-gadget
-$ make
+$ make build-ig
+$ sudo cp inspektor-gadget /usr/local/bin/kubectl-gadget
+$ kubectl gadget version
 ```
 
 Note:
-- if you wish to make changes to traceloop program, update `gadget-ds/gadget.Dockerfile` to pick your own image of traceloop.
-- if you wish to make other changes in the gadget container image, update `Makefile` to choose the default `gadgetimage`.
-
-See the [minikube](#Development-environment-on-minikube-for-the-traceloop-gadget) section for a faster development cycle.
+- the compilation uses `tools/image-tag` to choose the tag of the container
+image to use according to the branch that you are compiling.
+- you can push the docker images to another registry and use the `--image`
+argument to choose them as described below.
+- if you wish to make changes to traceloop program, update
+`gadget-ds/gadget.Dockerfile` to pick your own image of traceloop.
```

Comment:
I guess you will have to update gadget-ds -> gadget-container if your other PR gets in first.

Reply:
I propose this to go first, as the other have some conflicts and require a little bit more time.

Reply:
SGTM